packages/react-native-sdk/src/components/Participant/FloatingParticipantView/index.tsx

```diff
@@ -131,25 +143,27 @@ export const FloatingParticipantView = ({
           containerWidth={containerDimensions.width}
           initialAlignment={floatingAlignmentMap[alignment]}
         >
-          {ParticipantView && (
-            <ParticipantView
-              participant={participant}
-              videoMode={'video'}
-              style={[
-                styles.participantViewContainer,
-                style,
-                {
-                  backgroundColor: colors.static_grey,
-                  shadowColor: colors.static_black,
-                },
-                localParticipantsView.participantViewContainer,
-              ]}
-              // video z order must be one above the one used in grid view
-              // (which uses the default: 0)
-              videoZOrder={1}
-              {...participantViewProps}
-            />
-          )}
+          <Pressable onPress={onPressHandler}>
+            {ParticipantView && (
+              <ParticipantView
+                participant={participant}
+                videoMode={'video'}
+                style={[
+                  styles.participantViewContainer,
+                  style,
+                  {
+                    backgroundColor: colors.static_grey,
+                    shadowColor: colors.static_black,
+                  },
+                  localParticipantsView.participantViewContainer,
```

Comment:
I think we need to change the name `localParticipantsView` in this case, since floating view is not associated with local participant only.

Reply:
Oh damn, I forgot to do it when moving the LocalParticipantView to FloatingParticipantView back then. I have pushed the change, so it should be fixed now. Nice catch.